app/models/derived_variables/sales_log_variables.rb

```diff
@@ -75,7 +75,7 @@ def set_derived_fields!
     self.nationality_all = nationality_all_group if nationality_uk_or_prefers_not_to_say?
     self.nationality_all_buyer2 = nationality_all_buyer2_group if nationality2_uk_or_prefers_not_to_say?
 
-    self.numstair = 1 if is_firststair?
+    self.numstair = nil if is_firststair?
```

Comment:
I changed this to nil, because when changing the response to `firststair` from yes to no, you could not save the form, I think this is because the `numstair` question has a minimum value of 2. However, the acceptance criteria (AC) stated that `numstair` should be inferred as 1 when it is `firststair`. Could we approach this differently?